templates/default/container-executor.cfg.erb

```diff
@@ -1,2 +1,30 @@
+yarn.nodemanager.local-dirs=<%= node['hops']['tmp_dir'] %>/nm-local-dir
+yarn.nodemanager.log-dirs=<%= node['hops']['logs_dir'] %>/userlogs
 yarn.nodemanager.linux-container-executor.group=<%= @hops_group %>
 min.user.id=0
+banned.users=<%= node['hops']['hdfs']['user'] %>,<%= node['hops']['yarn']['user'] %>,<%= node['hops']['mr']['user'] %>,bin
```

Comment:
is the `bin` user here a typo?

Reply:
I copied the conf from the cloudera doc and they had the bin user in there so I thought it did not hurt to keep it: https://docs.cloudera.com/HDPDocuments/HDP3/HDP-3.1.5/data-operating-system/content/configure_yarn_for_running_docker_containers.html

Reply:
Ok